### ISSUE TITLE
Broken Icons

### ISSUE THREAD

**Issue by alivefromupstairs** [RemixDev/DRemixIssues#123]
**Describe the bug**
Icons are broken and represented by strings.

**To Reproduce**
In my case, just open.

**Screenshots**
![image](https://user-images.githubusercontent.com/46059092/70656846-206f7b00-1c5b-11ea-9f08-e7a5b9c925ce.png)

**Context:**
 - OS: Manjaro 18.1.4
 - Deez Version: 4.3.0 (4.2 too)
 - Did you build it from the repo?: No. packages from Mirrors (Mega) and AUR

**Comment by RemixDev** [comment]
You should try opening the app in chrome and check the console.
Let me know if you find any errors there

**Comment by aliveupstairs** [comment]
@RemixDev Could you provide (link to) instructions? should I build from repo?

**Comment by RemixDev** [comment]
Open the app and go to localhost:1730

**Comment by aliveupstairs** [comment]
Does nothing.
![image](https://user-images.githubusercontent.com/46059092/73495506-f57de980-43b6-11ea-9315-0e260ca017b8.png)

**Comment by RemixDev** [comment]
http://localhost:1730
Don't use https

**Comment by aliveupstairs** [comment]
> Don't use https

my bad
everything looks fine here
![localhost:1730 be looking good](https://user-images.githubusercontent.com/46059092/73496007-ed727980-43b7-11ea-8456-d0a4a03b6499.png)

**Comment by RemixDev** [comment]
Then there's something wrong with electron
I can't update to a newer version right now because of missing features
I have no idea on how to fix this issue

**Comment by aliveupstairs** [comment]
Yeah, it's near impossible if you can't reproduce it yourself and no errors. can I run `deezloader-remix` headlessly? I'm running chromium anyway.

**Comment by RemixDev** [comment]
Try using the -s argument, idk if it still works

**Comment by aliveupstairs** [comment]
It works. Are there any other useful flags? I tried `-h` and `--help`, they were ignored.

**Comment by RemixDev** [comment]
Just -s for now

**Comment by aliveupstairs** [comment]
> something wrong with electron

That's probably it. because if I run `localhost:1730` in [`surf`](https://surf.suckless.org/), a very minimal browser that lacks a lot of browser features, it looks the same.

![image](https://user-images.githubusercontent.com/46059092/73498672-39c0b800-43be-11ea-8d44-c84cc0ac231d.png)
